website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,195 @@
+also require significant planning, testing, and employee hours to execute
+successfully. 
+
+Customers who need assurances that their current installation will receive
+critical bug fixes and security patches with minimal service disruptions should
+consider moving to a Vault version with long-term support.
+
+<a id="what" />
+
+## What is long-term support?
+
+**Standard support** for Vault includes:
+
+- performance improvements.
+- feature updates and improvements.
+- bug fixes.
+- security patches.
+- CVE patches to correct vulnerabilities and exposures.
+
+Standard support covers "N&minus;2" versions, which means, at any given time,
+HashiCorp supports the current version ("N") and the two previous versions
+("N&minus;2").
+
+Vault versions typically update 3 times per calendar year, which means that
+standard support for a given Vault version lasts approximately 1 year:
+
+Version | Release date         | Standard support ends
+------- | -------------------- | ----------------------
+1.16    | CY24 Q1 (2024-03-27) | CY25 Q1 (1.19 release) 
+1.15    | CY23 Q3 (2023-09-27) | CY24 Q3 (1.18 release) 
+1.14    | CY23 Q2 (2023-06-21) | CY24 Q2 (1.17 release) 
+1.13    | CY23 Q1 (2023-03-01) | CY24 Q1 (2024-03-27) 
+1.12    | CY22 Q4 (2022-10-12) | CY23 Q3 (2023-09-27) 
+
+**Long-term support** extends support for specific Vault versions for an
+additional year with:
+
+- patches for bugs that may cause outages.
+- patches for critical vulnerabilities and exposures.
+- high-risk security patches.
+
+After the first year, LTS Vault versions move from standard support to long-term
+support for an additional year.
+
+<a id="where" />
+
+## Where do I enable long-term support?
+
+You do not need to download a separate binary or set a flag for LTS support. As
+long as you select an LTS Vault Enterprise version (e.g., 1.16, 1.19) when you
+[install](/vault/install) or [upgrade](/vault/docs/upgrading) your Vault
+instance, LTS is included.
+
+<a id="when" />
+
+## When are LTS versions released?
+
+As of Vault Enterprise 1.16, the first release of a calendar year includes
+long-term support.
+
+LTS versions overlap by one year with the previous LTS version entering its
+long-term support window when the new LTS version begins its standard support
+window.
+
+<a id="why" />
+
+## Why is there a risk to updating to a non-LTS Vault version?
+
+Long-term support is intended for Enterprise customers who cannot upgrade
+every quarter to stay within the standard support timeline of one year. The
+goal is to establish a predictable upgrade path with a longer timeline rather
+than extending the lifetime for every Vault version.
+
+Long-term support ensures your Vault version continues to receive critical
+patches for an additional year. If you upgrade to a non-LTS version, you are
+moving your Vault instance to a version that lacks extended support, which means
+your Vault version stops receiving updates once it leaves the standard support
```

Comment:
```suggestion
your Vault Enterprise version stops receiving updates once it leaves the standard support
```

Reply:
In general for this page, all language about standard support should state "Vault Enterprise" rather than "Vault"